tests/in/glsl/sampler2DArray.frag

```diff
@@ -0,0 +1,12 @@
+#version 440 core
```

Comment:
I don't think we should have a separate test per sampler type. Can we just call it `samplers.frag`?

Reply:
Done.